scmail.py

```diff
@@ -0,0 +1,205 @@
+import requests
+import click
+import pprint
+import sys
+import re
+from pathlib import Path
+from os import getlogin, environ
+import json
+from gpgopt import GpgOpt
+
+# Default to localhost if url is not given
+SECUREMAILBOX_URL = environ.get("SECUREMAILBOX_URL", "http://127.0.0.1:8080")
+
+@click.group()
+@click.pass_context
+def client(ctx):
+    """Initial gnupg."""
+    # Set config file path.
+    config_path = Path('scmail.conf')
+    ctx.obj['config'] = config_path
+
+    # If the config not exist, set user information. If not, run initial gpg.
+    if config_path.exists() is False:
+        gnupghome, email = ctx.invoke(set_user_info, gnupghome=click.prompt('Enter your gnupg home dir:', default='gnupgkeys'), email=click.prompt('Enter your email:', default=getlogin() + '@scmail.dev'))
+    else:
+        gnupghome, email = load_config()
+
+    ctx.obj['gpg'] = GpgOpt(mygnupghome=gnupghome, email=email)
+    # Initial pretty printer
+    ctx.obj['pp'] = pprint.PrettyPrinter(indent=2)
+
+
+@click.pass_context
+def load_config(ctx):
+    """Load config file."""
+    print("Now, laod previous config file:")
+    f = ctx.obj['config'].open('r').read()
+    config = re.split('\n|=', f)
+
+    for i in range(0, len(config), 2):
+        print('The {} is: {}'.format(config[i], config[i + 1]))
+
+    print('\n')
```

Comment:
This is functionally the same as just `print()`.

Reply:
Thanks, I changed it.